### PR TITLE
docs: Plugins.md

### PR DESCRIPTION
**Pull request by ghinks**
trivial grammar change removing **to**
```
We recommend to using the fastify-plugin
```

to

```
We recommend using the fastify-plugin
```

learning lots when re-reading the documentation , thank you.

### CODE REVIEW

File: docs/Plugins.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ You have two ways to tell Fastify to avoid the creation of a new context:
 - Use the [`fastify-plugin`](https://github.com/fastify/fastify-plugin) module
 - Use the `'skip-override'` hidden property
 
-We recommend to using the `fastify-plugin` module, because it solves this problem for you, and you can pass a version range of Fastify as a parameter that your plugin will support.
+We recommend using the `fastify-plugin` module, because it solves this problem for you, and you can pass a version range of Fastify as a parameter that your plugin will support.
```
Comment (kibertoad) [inline docs/Plugins.md:171]: Wording still feels off here. Wouldn't `you can pass a version range of Fastify that your plugin will support as a parameter` be more accurate in this context?

**Review by mcollina — APPROVED** [review]
lgtm

File: docs/Plugins.md (hunk 1 above)
Comment (ghinks) [reply docs/Plugins.md:171]: I agree, I was avoiding being too judgmental on my first PRs to the project. Let me re-word that in a different PR. This was just for the typo.

**Review by kibertoad — APPROVED** [review]

File: docs/Plugins.md (hunk 1 above)
Comment (ghinks) [reply docs/Plugins.md:171]: @kibertoad maybe something like 

When only adding decorators we recommend using the `fastify-plugin` module as it 
will not create a new scope. In addition `fastify-plugin` supports fastify versioning.

**Review by delvedor — APPROVED** [review]
LGTM

**Pull request merged**